HISTORY.md

```diff
@@ -2,6 +2,7 @@
 ## Unreleased
 ### Bug Fixes
 * Fixed the truncation error found in APIs/tools when dumping block-based SST files in a human-readable format. After fix, the block-based table can be fully dumped as a readable file.
+* Made `delayed_write_rate` more strictly enforced, except now avoiding unnecessary initial delays. Specifically, writes are only delayed if during elapsed time t they exceed the allowance for t + 1 millisecond.
```

Comment:
The first sentence is a little awkward, I think you mean "now avoiding" (no except).

Reply:
How about this:

> When hitting a write slowdown condition, no write delay (previously 1 millisecond) is imposed until `delayed_write_rate` is actually exceeded, with an initial burst allowance of 1 millisecond worth of bytes. Also, beyond the initial burst allowance, `delayed_write_rate` is now more strictly enforced, especially with multiple column families.